src/admin.py

```diff
@@ -178,6 +178,9 @@ def create_api_key(
     )
     create_user_api_key_in_db(db, new_api_key)
 
+    # Remove spaces and newlines returned in token
+    refresh_token = refresh_token.replace(" ", "").replace("\n", "")
```

Comment:
Wouldn't ```refresh_token.strip()``` accomplish the same thing? Also, are you sure this isn't an issue related to copy/pasting with the terminal? if the terminal width isn't large neough it'll likely split the token line when it's printed, regardless of whether you strip it before printing or not (at least that's the case in iTerm2 in macOS which is what i tried this on). The refresh_token variable returned here should not contain whitespaces or newlines.

Reply:
the intent of this was to be able to save the token to an environment variable and then add the contents of that environment variable to a config file. it added it to the config file with breaks and whitespace and as a result, was not valid.